zad5/app/views/tracks/index.html.erb

```diff
@@ -16,12 +16,17 @@
         <tr>
           <td><%= track.title %></td>
           <td class="text-center">
-            <%= track.artist.name %>
+            <%= link_to track.artist.name, track.artist %>
+          </td>
+          <td class="text-center">
+            <% track. playlists.each do |playlist| %>
+              <%= link_to playlist.title, playlist %>
+            <% end %>
           </td>
           <td class="text-center"></td>
           <td class="text-right">
-            <%= link_to 'edit', '#', class: 'btn btn-warning' %>
-            <%= link_to 'remove', '#', class: 'btn btn-danger'  %>
+            <%= link_to 'edit', edit_track_path(track), class: 'btn btn-danger' %>
+            <%= link_to 'remove', track_path(track), method: :delete, class: 'btn btn-danger' %>
```

Comment:
można dodać `data: { confirm: 'Are you sure?' }`, aby przypadkiem nie usunąć niczego 🙈